profiles/base/package.use.mask

```diff
@@ -7,6 +7,10 @@
 # This file is only for generic masks. For arch-specific masks (i.e.
 # mask everywhere, unmask on arch/*) use arch/base.
 
+# Jan Ziak <0xe2.0x9a.0x9b@gmail.com> (14 Mar 2018)
+# Mask local USE flag to satisfy repoman
```

Comment:
You should also unmask them in appropriate profiles where svgalib is available. Otherwise, there's no point in having the flag at all.